src/locators/defaultLocators.js

```diff
@@ -0,0 +1,47 @@
+var clientsidescripts = require('../scripts/clientsidescripts');
+var ByControlLocator = require('./byControlLocator');
+
+/**
+ * Provide jasmine locator
+ * @constructor
+ * @param {Config} config
+ * @param {Object} instanceConfig
+ * @param {Logger} logger
+ */
+function DefaultLocators(config,instanceConfig,logger){
+  this.config = config;
+  this.instanceConfig = instanceConfig;
+  this.logger = logger;
+}
+
+/**
+ * Register jasmine locator
+ * @param {By} by - jasmine By object on which to add the new locator
+ */
+DefaultLocators.prototype.register = function(by) {
+  this.logger.debug('Registering default locators');
+  // http://angular.github.io/protractor/#/api?view=ProtractorBy.prototype.addLocator
+
+  by.addLocator('jq', function(query,opt_parentElement) {
+    return $(opt_parentElement ? opt_parentElement + ' ' + query : query).toArray();
+  });
+
+  var byControlLocator = new ByControlLocator(clientsidescripts, this.logger);
+  byControlLocator.register(by, 'control');
+
+  /* example of how to register a new locator that:
```

Comment:
I  think it is better idea to have the new locator as a module that will use our byControl locator module.